__tests__/libs/ontouml2db/mapper.test.ts

```diff
@@ -0,0 +1,1595 @@
+/**
+ *
+ * Author: Gustavo Ludovico Guidoni
+ */
+
+import { GraphChecker } from '@libs/ontouml2db/graph/graph_tester/GraphChecker';
+import { NodeChecker } from '@libs/ontouml2db/graph/graph_tester/NodeCheker';
+import { PropertyChecker } from '@libs/ontouml2db/graph/graph_tester/PropertyChecker';
+import { TrackerChecker } from '@libs/ontouml2db/graph/graph_tester/TrackerChecker';
+import { RelationshipChecker } from '@libs/ontouml2db/graph/graph_tester/RelationshipChecker';
+import { Transformation2DB } from '@libs/ontouml2db/Transformation2DB';
+import { Cardinality } from '@libs/ontouml2db/graph/util/enumerations';
+import { RunExample } from '@test-models/transformation';
+import { test_001 } from '@test-models/transformation';
+import { test_002 } from '@test-models/transformation';
+import { test_003 } from '@test-models/transformation';
+import { test_004 } from '@test-models/transformation';
+import { test_005 } from '@test-models/transformation';
+import { test_006 } from '@test-models/transformation';
+import { test_007 } from '@test-models/transformation';
+import { test_008 } from '@test-models/transformation';
+import { test_009 } from '@test-models/transformation';
+import { test_010 } from '@test-models/transformation';
+import { test_011 } from '@test-models/transformation';
+import { test_012 } from '@test-models/transformation';
+import { test_013 } from '@test-models/transformation';
+import { test_014 } from '@test-models/transformation';
+import { test_015 } from '@test-models/transformation';
+import { test_016 } from '@test-models/transformation';
+import { test_017 } from '@test-models/transformation';
+import { test_018 } from '@test-models/transformation';
+import { test_019 } from '@test-models/transformation';
+import { test_020 } from '@test-models/transformation';
+import { test_021 } from '@test-models/transformation';
+import { test_022 } from '@test-models/transformation';
+import { test_023 } from '@test-models/transformation';
+import { test_024 } from '@test-models/transformation';
+import { test_025 } from '@test-models/transformation';
+import { test_026 } from '@test-models/transformation';
+import { test_027 } from '@test-models/transformation';
+import { test_028 } from '@test-models/transformation';
+
+describe('Database transformation test', () => {
```

Comment:
This test file is just too big. Could you break it into smaller test files? Maybe grouping them by theme (e.g. what they test)?